tests/cli_test.py

```diff
@@ -26,6 +27,20 @@ def test_options(self):
         self.assertRaises(SystemExit, main)
         sys.argv = ["python", "info", "does_not_exist"]
         self.assertRaises(SystemExit, main)
+        sys.argv = ["python", "info", "--gs", "greyscale.png"]
```

Comment:
good, I like more coverage. Maybe we could add messages in the asserts to explain why we expect those checks to fails (e.g., number-of-plates 101 should fail because the limit is to 100)